src/explore-education-statistics-common/src/modules/charts/components/utils/getReferenceLineLabelPosition.ts

```diff
@@ -0,0 +1,47 @@
+import { Axis, AxisType } from '@common/modules/charts/types/chart';
+import { AxisDomain, ViewBox } from 'recharts';
+
+export default function getReferenceLineLabelPosition({
+  axis,
+  axisType,
+  otherAxisDomain = [0, 0],
+  otherAxisPosition,
+  viewBox,
+}: {
+  axis: Axis;
+  axisType: AxisType;
+  otherAxisDomain?: [AxisDomain, AxisDomain];
+  otherAxisPosition?: number;
+  viewBox?: ViewBox;
+}) {
```

Comment:
Would specify an explicit return type like:
```tsx
interface LabelPosition {
  x: number;
  y: number;
}
```

Note that I'm also suggesting we simplify the property names for brevity. No need to postfix these with 'position'.